DmitriySkakun/HW4/src/app/components/Post.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+// import ReactDom from 'react-dom';
+// import './app/styles/style.css';
+
+class Post extends React.Component{
+    render(){
+        console.log(this.props.children);
```

Comment:
```suggestion
        const {title, postBody, author} = this.props;
```
вот такая фишка с деструктуризацией)